Tests/regressionTests.js

```diff
@@ -0,0 +1,301 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
```

Comment:
Is it required to use var fo every line, can you use it only for first:
var a = ...,
      b = ...,

Reply:
I added this type of declaration